### PR TITLE
Don 918 update homepage

### PR DESCRIPTION
**Pull request by bdsl**
To test, with a simulated current date, visit a URL such as http://localhost:4200/?simulatedDate=2023-11-28T12:00:00%2B00:00

The number at the end after `%2B` is the timezone offset from UTC - so keep it at zero if you just want to test for London in winter.

image

image

image

image

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -38,19 +21,60 @@ export class HomeComponent implements OnInit {
...
+        text: 'See Results',
+        href: new URL('/christmas-challenge-2023', environment.donateGlobalUriPrefix)
       }
     },
-    globalCard,
     {
+      appearAt: 'asap',
+      disappearAt: GMFApplicationCloseDate,
+      headerText: "Applications for Green Match Fund are now open!",
+      bodyText: "Apply by January 16 2024",
```
Comment (NoelLH) [inline src/app/home/home.component.ts:67]: Should we standardise on a date format for these? Some copy has the number with suffix followed by the month

**Review by NoelLH — APPROVED** [review]
Added a small question about dates in copy, but the logic all looks good to me

File: src/app/home/home.component.ts (hunk 1 above)
Comment (bdsl) [reply src/app/home/home.component.ts:67]: Yes we probably should standardise, either to have the suffix or not have it. I don't have a strong feeling which way, I think I'd rather we leave it out everywhere. @BethBigGive what do you think?

Comment (bdsl) [reply src/app/home/home.component.ts:67]: Merged so will move question to ticket.

**Pull request merged**